lib/logstash/outputs/elasticsearch/http_client.rb

```diff
@@ -133,7 +133,7 @@ def join_bulk_responses(bulk_responses)
     end
 
     def bulk_send(body_stream)
-      params = upload_compression ?  {:headers => {"Content-Encoding" => "gzip"}} : {}
+      params = http_compression ?  {:headers => {"Accept-Encoding" => "gzip,deflate", "Content-Encoding" => "gzip"}} : {}
```

Comment:
Yes, response compression is on by default. Manticore sets the Accept-Encoding header already and elastic can (and should, but if it doesn't it's an es lack of feature) respond with gzip encoded data.